optima/results.py

```diff
@@ -102,7 +102,7 @@ def __repr__(self):
 
 class Resultset(object):
     ''' Structure to hold results '''
-    def __init__(self, raw=None, name=None, simpars=None, project=None, settings=None, data=None, parset=None, progset=None, budget=None, budgetyears=None, domake=True):
+    def __init__(self, name=None, raw=None, simpars=None, project=None, settings=None, data=None, parset=None, progset=None, budget=None, coverage=None, budgetyears=None, domake=True):
```

Comment:
i know it's inconsistent, but i had it this way because usually you _don't_ want to specify a name for a resultset, and the other syntax meant that you could do Resultset(raw), which is all you actually need -- so here i agree it would be more transparent if raw were _not_ a kwarg, except then you couldn't do the syntax of Resultset(name='foo', raw=raw) as you would with other objects -- so i'd prefer to leave it as it was